docs/introduction-to-smart-contracts.rst

```diff
@@ -565,3 +565,21 @@ contracts, the Ether is forever lost.
 If you want to deactivate your contracts, you should instead **disable** them
 by changing some internal state which causes all functions to revert. This
 makes it impossible to use the contract, as it returns Ether immediately.
+
+
+.. index:: ! precompiled contracts, precompiles, contract;precompiled
+
+.. _precompiledContracts:
+
+Precompiled Contracts
+=====================
+
+There is a small set of contract addresses that are special:
+The address range between `1` and (including) `8` contain
+"precompiled contracts" that can be called as any other contract
+but their behaviour (and their gas consumption) is not defined
+by EVM code stored at that address (they do not contain code)
+but instead is implemented in the EVM execution environment itself.
+
+It could be that further precompiled contracts can be added in the future,
+but their addresses will always be small.
```

Comment:
```suggestion
but their addresses will always be small integers.
```

Reply:
This implies that addresses are integers. Maybe `small (as integers).`?

Reply:
I see that you just added the range, which makes my comment moot :)